include/links-permalinks.php

```diff
@@ -62,6 +62,41 @@ public function __construct( &$model ) {
 		$this->use_trailing_slashes = ( '/' == substr( $permalink_structure, -1, 1 ) );
 	}
 
+	/**
+	 * Initializes permalinks.
+	 *
+	 * @since 3.5
+	 *
+	 * @return void
+	 */
+	public function init() {
+		if ( did_action( 'wp_loaded' ) ) {
+			$this->can_prepare_rewrite_rules();
+		} else {
+			add_action( 'wp_loaded', array( $this, 'can_prepare_rewrite_rules' ), 9 ); // Just before WordPress callback `WP_Rewrite::flush_rules()`.
+		}
+	}
+
+	/**
+	 * Fires our own action telling Polylang plugins
+	 * and third parties are able to prepare rewrite rules.
+	 *
+	 * @since 3.5
+	 *
+	 * @return void
+	 */
+	public function can_prepare_rewrite_rules() {
```

Comment:
This method name makes no sense to me 😅
Since it starts with `can_`, I expect it to return a boolean 😖

```
init()
prepare_rewrite_rules()
can_prepare_rewrite_rules()
remove_hooks()
```

Am I the only one being confused?

Reply:
Indeed, it's a bit confusing.
What do you suggest?

Reply:
For instance:
```php
init();
prepare_rewrite_rules();
fire_prepare_rewrite_rules();
unprepare_rewrite_rules();
```
?

Reply:
Feels better indeed.

Reply:
Done in f8a2ee95.